sponsors2016.php

```diff
@@ -125,7 +125,11 @@
 	<h2 style="text-align:center">Fine Arts Photography</h2>
 	<div class="row">
 	<div class=""><a href="" target="_blank" >
+<<<<<<< HEAD
```

Comment:
I am fixing the conflict this time. From next time, make sure that conflicts don't make it into Pull Requests.